cmd/singularity/cli_test.go

```diff
@@ -52,7 +52,7 @@ func run(m *testing.M) int {
 	cmdPath = path
 
 	// Ensure config is installed
-	if fi, err := os.Stat(files.GetSysConfigFile()); err != nil {
+	if fi, err := os.Stat(syfs.GetSysConfigFile()); err != nil {
```

Comment:
Same here, `buildcfg.SINGULARITY_CONF_FILE` can be used.

Reply:
> Same here, `buildcfg.SINGULARITY_CONF_FILE` can be used.

it can be used everywhere, yes, because syfs.GetSysConfigFile() is simply "return buildcfg.SINGULARITY_CONF_FILE".